utility/serial.cpp

```diff
@@ -92,6 +92,53 @@ void ArdusatSerial::begin(unsigned long baud, bool setXbeeSpeed)
   }
 }
 
+/**
+ * Begin serial communications with a Sparkfun BlueSMiRF module at the specified baud
+ * rate.
+ *
+ * Note that baud rates above ~57600 are not well-supported by SoftwareSerial, and 
+ * even 57600 may cause some bugs.
+ *
+ * @param baud rate for serial communications
+ */
+void ArdusatSerial::beginBluetooth(unsigned long baud)
+{
+  if (_mode == SERIAL_MODE_HARDWARE || _mode == SERIAL_MODE_HARDWARE_AND_SOFTWARE) {
+    bool valid = true;
+    if (baud != 9600) {
+      _soft_serial->end();
+      _soft_serial->begin(9600); // shipping bluesmirf with default of 9600
```

Comment:
I'd be in favor of us setting them before we ship the modems rather than doing temp ones and having to update it each time the device boots.  95%+ of our customers will leave it as the default.